infra/notifications/package.json

```diff
@@ -26,6 +26,7 @@
   "author": "Origin Protocol Inc",
   "license": "MIT",
   "dependencies": {
+    "@origin/auth-server": "^0.1.0",
```

Comment:
Why does this depend on the auth-server, rather than the auth-client?

Reply:
I could probably move the middleware to a different package. But that would still need DB and other dependencies it has now to verify the token. The other way I could think for this would be to do a HTTP request. I'm not sure how good it is to do that in a middleware though